src/ipfs.io/docs/getting-started/index.html

```diff
@@ -142,7 +151,7 @@
 gateway, and the gateway sent it to your browser.
 
 <div class="alert alert-warning">
-Note: depending on the state of the network, the `curl` may take a while. The public gateways may be overloaded or having a hard time reaching you.
+Note: depending on the state of the network, the `curl` may take a while. The public gateways may be overloaded or having a hard time reaching you. If you use curl, then you get:<br/>curl: (7) couldn't connect to host</3>
```

Comment:
the errors will vary depending on curl version, and what is actually going wrong. best to leave the error off.